### ISSUE TITLE
List usernames following a user

### ISSUE THREAD

**Issue by romainpiel** [appdotnet/api-spec#232]
I would really need a request for the app I'm currently building. The same function as:

```
https://alpha-api.app.net/stream/0/users/[user_id]/followers/ids
```

But returning a list of the usernames.

Would it be possible?

**Comment by mattflaschen** [comment]
The same thing would help me solve a problem in Noodle.  But rather than adding more resources, I'd like to see a good solution to issue #96 (getting only subsets of information).

That would solve this problem (just specify the subset as username, id, or username+id, etc.) and add flexibility in general.

**Comment by berg** [comment]
We won't be able to do this as a standalone thing, unfortunately. It'll have to be solved under #96. I'll comment there too.
